Archive to file

RonanXin opened the pull request:
Save Order:
Date, Time,
Contamination per sample,
Points>0.1,
Points>0.4,
Points>1.0,
Number_per_L,,
Grams_per_L,
Histogram,
EquivD,
Area,

RonanXin committed:
Archive to file

Save Order:
Date, Time,
Contamination per sample,
Points>0.1,
Points>0.4,
Points>1.0,
Number_per_L,,
Grams_per_L,
Histogram,
EquivD,
Area,

byocp commented:
@RonanXin lets not spend time on C# for now, and make the MATLAB GUI and Particle Counter Engine become turn-key ready for Austria and then we start working on the C#. What do you think?

RonanXin committed:
Otsu's method

Otsu's method for threshold calculation is optional, and the manually
set value can be set in config.txt
